File: Programmers/level1/strangeString/strange_string.py
# 문제 출처 : https://programmers.co.kr/learn/courses/30/lessons/12930

def solution(s):
    answer = ""
    cnt = 0
    for spell in s:
        if cnt % 2 != 0:
            answer += spell.lower()
            cnt += 1
        elif spell == " ":
            answer += spell
            cnt = 0
        else:
            answer += spell.upper()
            cnt += 1
    print(answer)
    return answer

File: Programmers/level1/strangeString/strangeString.js
// https://programmers.co.kr/learn/courses/30/lessons/12930?language=javascript

export const solution = (s) => {
    let answer = '';
    let idx = 0;
    for(let i = 0; i < s.length; i++){
        if(s[i] === ' '){
            answer += ' ';
            idx = 0;
        } else if(idx % 2 === 0){
            answer += s[i].toUpperCase();
            idx += 1;
        } else {
            answer += s[i].toLowerCase();
            idx += 1;
        }
    }
    return answer;
}